### PR TITLE
deflake web tests option 1: one `flutter test` per test

### PR DESCRIPTION
**Pull request by yjbanov**
## Description

Run one `flutter test` command per test file so we do not share Chrome instances between multiple tests.

## Related Issues

https://github.com/flutter/flutter/issues/47003

### CODE REVIEW

File: packages/flutter_tools/lib/src/build_runner/web_compilation_delegate.dart [hunk 1]
```diff
@@ -203,15 +204,34 @@ class BuildDaemonCreator {
...
+      if (!globals.fs.isFileSync(buildScriptPackages)) {
+        await pub.get(
+          context: PubContext.pubGet,
+          directory: globals.fs.file(buildScriptPackages).parent.path,
+        );
+      }
+
+      final ProcessResult snapshotResult = await globals.processManager.run(<String>[
+        globals.artifacts.getArtifactPath(Artifact.engineDartBinary),
+        '--snapshot=$snapshot',
```
Comment (jonahwilliams) [inline packages/flutter_tools/lib/src/build_runner/web_compilation_delegate.dart]: `--snapshot-kind=app-jit` ? or does that not make a big difference

Comment (yjbanov) [reply packages/flutter_tools/lib/src/build_runner/web_compilation_delegate.dart]: I'll give it a shot. Heads up, this PR is not ready for review yet. I'll need to chat with you about how to properly cache and invalidate the snapshot.

**Comment by skia-gold** [comment]
Gold has detected one or more untriaged digests on patchset 6.
Please triage them at https://flutter-gold.skia.org/search?issue=52495.

**Comment by jonahwilliams** [comment]
Still working on this @yjbanov ?

**Comment by jonahwilliams** [comment]
Because of the lack of activity on this PR I'm going to close it to remove from our review queue. Please feel free to send us a new PR or discuss in an issue anytime.

**Pull request closed without merging**